src/osiris.erl

```diff
@@ -133,13 +134,14 @@ start_replica(Replica, Config) ->
 write(Pid, WriterId, Corr, Data) ->
     osiris_writer:write(Pid, self(), WriterId, Corr, Data).
 
--spec write_tracking(pid(), binary(), offset()) -> ok.
-write_tracking(Pid, TrackingId, Offset) ->
-    osiris_writer:write_tracking(Pid, TrackingId, Offset).
+-spec write_tracking(pid(), binary(), {tracking_type(), offset() | timestamp()}) -> ok.
+write_tracking(Pid, TrackingId, {_TrkType, _TrkData} = Tracking) ->
```

Comment:
This will break code that uses it in rabbit

Reply:
Pushed a commit to retain backwards compatibility.